src/python/pants/backend/codegen/grpcio/grpcio_run.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import functools
+import logging
+from collections import OrderedDict
+
+from pants.backend.codegen.grpcio.grpcio_prep import GrpcioPrep
+from pants.backend.codegen.grpcio.python_grpcio_library import PythonGrpcioLibrary
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.base.build_environment import get_buildroot
+from pants.base.exceptions import TaskError
+from pants.base.workunit import WorkUnitLabel
+from pants.task.simple_codegen_task import SimpleCodegenTask
+from pants.util.contextutil import pushd
+
+
+class GrpcioRun(SimpleCodegenTask):
```

Comment:
This task will only generate grpc for python, correct? If so, this should probably follow the naming conventions that are used for thrift and protobuf. That would mean that the package would be something like: `pants/backend/codegen/grpcio/python`.

Reply:
^ this is still an issue.

Reply:
I've moved code to `python` package, as suggested.